src/amo/components/AddonMeta/index.js

```diff
@@ -47,6 +48,40 @@ export class AddonMetaBase extends React.Component<Props> {
     } else {
       reviewTitle = i18n.gettext('No Ratings');
     }
+    const overallRating = i18n.gettext('Overall Rating');
+
+    const makeLink = (content) => {
+      if (addon) {
+        return (
+          <Link
+            className="Addon-all-reviews-link"
+            to={`/addon/${addon.slug}/reviews/`}
+          >
+            {content}
+          </Link>
+        );
+      }
+      return content;
+    };
+
+    const ratingLink = (content) => {
```

Comment:
This function is the same as the `makeLink` function, except that instead of outputting the `content` variable inside the `<Link>` it outputs a `<Rating>` component. 

It seems like you could use the `makeLink` function for this too, and just pass the `<Rating>` component into it as `content`. Do you want to give that a try?